### DISCUSSION TITLE
How do I integrate my remote?

### DISCUSSION THREAD

**Discussion by SigmaKlim** [Holzhaus/sphinx-multiversion#124] [Q&A]
Hi! I am pretty new to sphinx and I cannot figure out how I should let sphinx know about my remote (branches and tags), so that multiversion would work.
My documentation root directory (containing source and build folders) is where my local repo is (it has .git folder there). I reproduce all the steps from quickstart and get no versions in the sidebar.  For debug, I added "else" part to the sample template to see whether it runs and any versions are detected. I can see that the code from versioning.html runs, but no versions are detected by sphinx.
Could you guide me, please?

**Comment by BurningEnlightenment** [comment]
Hi, please post your conf.py, provide a reprex (reproducing example) or if possible a [MCVE](https://stackoverflow.com/help/minimal-reproducible-example).

Generally speaking have you run `sphinx-multiversion` instead of `sphinx-build`?
